README.md

```diff
@@ -286,6 +313,12 @@ Set GPIO activeLow setting.
 Reverse the effect of exporting the GPIO to userspace. A Gpio object should not
 be used after invoking its unexport method.
 
+##### static accessible
+Returns true if the system has access to the basic subsystem onoff utalizes.
```

Comment:
The description "Returns true if the system has access to the basic subsystem onoff utalizes" doesn't describe what `accessible` does. If false is returned access to GPIO may still be possible but sudo privileges may be required for access.
What system is being referred to here?
What subsystem is being referred to here?